airflow/jobs/scheduler_job_runner.py

```diff
@@ -705,12 +705,14 @@ def _process_executor_events(self, session: Session) -> int:
             ti_primary_key_to_try_number_map[ti_key.primary] = ti_key.try_number
 
             self.log.info(
-                "Executor reports execution of %s.%s run_id=%s exited with status %s for try_number %s",
+                "Executor received event with state %s for task instance "
+                "dag_id=%s task_id=%s run_id=%s try_number=%s map_index=%s",
+                state,
                 ti_key.dag_id,
                 ti_key.task_id,
                 ti_key.run_id,
-                state,
```

Comment:
Do we not want state here?

Reply:
we do, good catch thanks

Reply:
oh, i did not drop it, it was just moved up :)